packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts

```diff
@@ -228,15 +212,21 @@ async function waitForSummaryOp(container: IContainer): Promise<boolean> {
 	});
 }
 
-describeNoCompat("Summarizer with local data stores", (getTestObjectProvider) => {
+describeNoCompat("Summarizer with local changes", (getTestObjectProvider) => {
 	let provider: ITestObjectProvider;
 
-	beforeEach(async () => {
+	beforeEach(async function () {
 		provider = getTestObjectProvider({ syncSummarizer: true });
+
+		// These tests validate client logic. Testing against multiple services won't make a difference.
+		if (provider.driver.type !== "local") {
+			this.skip();
+		}
 		settings = [];
```

Comment:
?
```suggestion
		settings = {};
```

Reply:
haha yeah I noticed this too. It works apparently!

Reply:
haha yeah JS for the win